run-integration-tests.py

```diff
@@ -495,10 +533,10 @@ def __exit__(self, tpe, value, traceback):
         action="store_true",
         help="Run the DynamoDB integration tests (and only them)")
     parser.add_argument(
-        "--dbb-packages",
+        "--packages",
```

Comment:
`--dbb-packages` is likely mentioned in some invocation examples for this command, update them